Examples/WildeGuess/WildeGuess/Components/MonochromeQuoteComponent.mm

```diff
@@ -15,16 +15,30 @@
 #import "QuoteContext.h"
 
 @implementation MonochromeQuoteComponent
+{
+  NSString *_text;
+  QuoteContext *_context;
+}
 
 + (instancetype)newWithText:(NSString *)text context:(QuoteContext *)context
+{
+  auto const c = [super new];
+  if (c) {
+    c->_text = text;
+    c->_context = context;
+  }
+  return c;
```

Comment:
This is a good example of the additional boilerplate required.

Reply:
Yep, I tried to fix all this with templates/code-gen before, but couldn't get it to be minimal.